barretenberg/cpp/src/barretenberg/client_ivc/client_ivc.cpp

```diff
@@ -153,7 +153,12 @@ std::pair<ClientIVC::PairingPoints, ClientIVC::TableCommitments> ClientIVC::
         break;
     }
     case QUEUE_TYPE::PG_FINAL: {
-        // Constuct the hiding circuit
+        // we should construct the hiding kernel in this case
```

Comment:
The original comment `// Construct the hiding circuit` is more clear. "we should" makes it sound like a TODO